outputs/client.go

```diff
@@ -162,6 +162,9 @@ func (c *Client) Post(payload interface{}) error {
 		if c.OutputType == Kubeless {
 			body, _ := ioutil.ReadAll(resp.Body)
 			log.Printf("[INFO]  : Kubeless - Function Response : %v\n", string(body))
+		} else if c.OutputType == Openfaas {
+			body, _ := ioutil.ReadAll(resp.Body)
```

Comment:
you can move this line before the `if` for avoiding duplication of code

Reply:
done

Reply:
done